driver.sh

```diff
@@ -0,0 +1,78 @@
+set -ux
+
+check_dependencies() {
+  set -e
+
+  test -x `which nproc`
+  test -x `which gcc`
+  test -x `which aarch64-linux-gnu-as`
+  test -x `which aarch64-linux-gnu-ld`
+  test -x `which qemu-system-aarch64`
+  test -x `which timeout`
+  test -x `which unbuffer`
+  test -x `which clang-8`
+
+  set +e
+}
+
+mako_reactor() {
+  make -j$(nproc) $@
+}
+
+build_linux() {
+  local clang=$(which clang-8)
+
+  cd linux
+  export ARCH=arm64
+  export CROSS_COMPILE=aarch64-linux-gnu-
+  # TODO: do we really want to be building the kernel from scratch every time?
+  # Rerunning ninja above will not rebuild unless source files change.
+  #make CC=$clang mrproper
+  mako_reactor CC=$clang defconfig
+  mako_reactor CC=$clang HOSTCC=$clang
+  cd -
+}
+
+build_root() {
+  if [[ ! -d buildroot ]]; then
+    wget https://buildroot.org/downloads/buildroot-2018.08.tar.gz
+    tar -xzf buildroot-2018.08.tar.gz
+    mv buildroot-2018.08 buildroot
+    rm -rf buildroot-2018.08.tar.gz
+  fi
+
+  if [[ ! -e buildroot/output/images/rootfs.ext4 ]]; then
+    mkdir -p buildroot/overlays/etc/init.d/
+    cp -f inittab buildroot/overlays/etc/.
+    cp -f S50yolo buildroot/overlays/etc/init.d/.
+
+    cd buildroot
+    make defconfig BR2_DEFCONFIG=../buildroot.config
+    mako_reactor
```

Comment:
:man: even buildroot takes too long.  Might have to make these prebuilt images, too...

Reply:
I've uploaded prebuilts in https://github.com/ClangBuiltLinux/continuous-integration/commit/aeea4e2b3c3e85adcee912ebdd8182fc6dffeb5f